src/main/java/io/ebean/dbmigration/ddlgeneration/platform/SqlServerDdl.java

```diff
@@ -103,16 +103,19 @@ public String createSequence(String sequenceName, int initialValue, int allocati
   @Override
   public String alterColumnDefaultValue(String tableName, String columnName, String defaultValue) {
 
-    if (isDropDefault(defaultValue)) {
-      return "-- alter table " + tableName + " drop constraint <unknown>  -- find the appropriate constraint for default value on column " + columnName;
+    if (DdlHelp.isDropDefault(defaultValue)) {
+      return "alter table " + tableName + " drop constraint df_" + tableName + "_" + columnName;
```

Comment:
constraints for sqlserver must be named (Will add functionality to DbConstraintNaming later)